integration/GET_KERNEL_QEMU

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# This script is intended to run, locally, the tests we run at circleci,
+# precisely as they are run there.
+#
+# to do so, it:
+# o creates a directory to store local artifacts retrieved from docker
+#   see TMP= below
+# o runs a the standard test container to retrieve a the qemu, kernel, and bios image
+# o runs go test with a default set of tests (./...)
+#
+# NOTE: if you want more complex behavior, don't make this script more
+# complex. Convert it to Go. Complex shell scripts suck.
+
+# These docker artifacts should not persist. Place them in tmp.
+# tmp is in .gitignore
+# I would prefer /tmp/$$.
+# Docker really doesn't like this for some reason, even when
+# I map it to /out inside the container.
+# TMP=/tmp/$$
+TMP=`pwd`/tmp
+mkdir -p $TMP
+chmod 777 $TMP
+
+# The default value is AMD64, but you can override it, e.g.
+# UROOT_TESTARCH=arm64 bash RUNLOCAL
+export ${UROOT_TESTARCH:=amd64}
```

Comment:
Shouldn't this be:

```
export UROOT_TESTARCH=${UROOT_TESTARCH:=amd64}
```

Reply:
it's not how it's commonly done. It works, I'd rather not touch it.

Reply:
oh, oops, I got it backwards, and yet ... it works. Still. Will fix.